css/css-box/margin-trim/grid-inline-start-001-ref.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>margin-trim: grid-inline-start-001</title>
```

Comment:
This file looks like duplicate cruft?

This is the only "-001" numbered file in this patch. And the patch includes a version of this file without the -001 suffix... (the immediate next file in this patch).